demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -461,33 +461,25 @@ def report_failed_lint_checks(self, lint_status: dict, pkgs_status: dict, return
             pkgs_status(dict): All pkgs status dict
             return_exit_code(int): exit code will indicate which lint or test failed
         """
-        for check in ["flake8", "XSOAR_linter", "bandit", "mypy", "vulture"]:
+        for check in ["flake8", "XSOAR_linter", "bandit", "mypy", "vulture", "pylint", "pwsh_analyze", "pwsh_test"]:
             if EXIT_CODES[check] & return_exit_code:
-                sentence = f" {check.capitalize()} errors "
-                print(f"\n{Colors.Fg.red}{'#' * len(sentence)}{Colors.reset}")
-                print(f"{Colors.Fg.red}{sentence}{Colors.reset}")
-                print(f"{Colors.Fg.red}{'#' * len(sentence)}{Colors.reset}\n")
-                for fail_pack in lint_status[f"fail_packs_{check}"]:
-                    print(f"{Colors.Fg.red}{pkgs_status[fail_pack]['pkg']}{Colors.reset}")
-                    print(pkgs_status[fail_pack][f"{check}_errors"])
-                    self.linters_error_list.append({
-                        'linter': check,
-                        'pack': fail_pack,
-                        'type': 'error',
-                        'messages': pkgs_status[fail_pack][f"{check}_errors"]
-                    })
-
-        for check in ["pylint", "pwsh_analyze", "pwsh_test"]:
-            check_str = check.capitalize().replace('_', ' ')
-            if EXIT_CODES[check] & return_exit_code:
-                sentence = f" {check_str} errors "
+                sentence = f" {check.capitalize().replace('_', ' ')} errors "
                 print(f"\n{Colors.Fg.red}{'#' * len(sentence)}{Colors.reset}")
                 print(f"{Colors.Fg.red}{sentence}{Colors.reset}")
                 print(f"{Colors.Fg.red}{'#' * len(sentence)}{Colors.reset}\n")
                 for fail_pack in lint_status[f"fail_packs_{check}"]:
                     print(f"{Colors.Fg.red}{fail_pack}{Colors.reset}")
-                    for image in pkgs_status[fail_pack]["images"]:
-                        print(image[f"{check}_errors"])
+                    if pkgs_status[fail_pack].get(f"{check}_errors", None):
+                        print(pkgs_status[fail_pack][f"{check}_errors"])
+                        self.linters_error_list.append({
+                            'linter': check,
+                            'pack': fail_pack,
+                            'type': 'error',
+                            'messages': pkgs_status[fail_pack][f"{check}_errors"]
+                        })
```

Comment:
probably ok but let's make sure this change does not affect the way we output the json file report.

We need to test a run with `-j` flag for this.

Reply:
ok will check